css/style.css

```diff
@@ -381,6 +393,8 @@ footer {
       justify-content: space-between; } }
   @media (min-width: 768px) {
     footer ul {
+      -webkit-box-orient: horizontal;
+      -webkit-box-direction: normal;
       -webkit-flex-flow: row;
       -ms-flex-flow: row;
       flex-flow: row;
```

Comment:
this file need's to be automatically generated by the Sass compiler, we can't write our code directly here